openedx/core/djangoapps/edx_global_analytics/utils/utilities.py

```diff
@@ -5,11 +5,25 @@
 import calendar
 import httplib
 import logging
-from datetime import date, timedelta
-
 import requests
 
-from openedx.core.djangoapps.edx_global_analytics.utils.cache_utils import cache_instance_data, get_query_result
+from datetime import date, timedelta
```

Comment:
Please resort imports. Now it's wrong.
You can use 'isort' in the terminal on your local env.